R/zzz.R

```diff
@@ -0,0 +1,7 @@
+.onLoad <- function(libname, pkgname) {
```

Comment:
Can we rename this file to R/utils_onLoad.R?

Reply:
done